expr/functions/asPercent/function.go

```diff
@@ -86,6 +86,7 @@ func (f *asPercent) Do(ctx context.Context, e parser.Expr, from, until int64, va
 			return nil, types.ErrWildcardNotAllowed
 		}
 		if len(total) == 1 {
+			total = helper.AlignSeries(append(total, arg...))[0:1]
```

Comment:
I think this is a left-over from the other PR you've opened:
https://github.com/go-graphite/carbonapi/pull/500

Could you please remove that part from this PR?

Reply:
Yep, wrong rebase. Thanks